### PR TITLE
New update to GHGA Site: News and Events

### PR DESCRIPTION
**Pull request by eufjan**

### CODE REVIEW

File: assets/locales/en/translation.json [hunk 1]
```diff
@@ -27,15 +27,39 @@
...
       "title": "February 12, 2020 - GHGA.de Website Goes Online",
       "content": "The German Human Genome-Phenome Archive (GHGA), which will be federated with the European Genome-Phenome Archive (EGA), and which is designed to archive and facilitate analysis of human access controlled research data from Germany, has gone live with a pilot version, distributing research data from Heidelberg and Tübingen. We look forward to its use by human biology, genetics and disease research communities."
     }
   },
+  "events": {
+    "navTitle": "Events",
+    "title": "Events",
+    "item1": {
+      "title": "December 10, 2020 - GHGA Virtual Kick-Off Event",
+      "content": "On December 10, 2020 all GHGA Participant will meet virtually to discuss the initiation and first steps of the consortium. Further information and registration is available via this <a href=\"https://indico.dkfz.de/e/ghga-kickoff\" target=\"_blank\">link</a>."
```
Comment (vinjana) [inline assets/locales/en/translation.json:56]:
```suggestion
      "content": "On December 10, 2020 all GHGA participants will meet virtually to discuss the initiation and first steps of the consortium. Further information and registration is available via this <a href=\"https://indico.dkfz.de/e/ghga-kickoff\" target=\"_blank\">link</a>."
```

File: assets/locales/en/translation.json [hunk 2]
```diff
@@ -27,15 +27,39 @@
...
+    },
+	"item2": {
+	"title": "GHGA Events Calendar",
+	"content": "An overview of all GHGA Events can found at this <a href=\"https://calendar.google.com/calendar/embed?src=cu680i1cppr6446qk50059fs78%40group.calendar.google.com&ctz=Europe%2FBerlin\" target=\"_blank\">link</a>."
```
Comment (vinjana) [inline assets/locales/en/translation.json:60]:
```suggestion
	"content": "An overview of all GHGA events can found at this <a href=\"https://calendar.google.com/calendar/embed?src=cu680i1cppr6446qk50059fs78%40group.calendar.google.com&ctz=Europe%2FBerlin\" target=\"_blank\">link</a>."
```

File: assets/locales/en/translation.json [hunk 3]
```diff
@@ -27,15 +27,39 @@
...
+      "title": "July 30, 2020 - Interview with Oliver Kohlbacher and Oliver Stegle on the aims of GHGA",
+      "content": "Oliver Stegle and Oliver Kohlbacher have talked to the <a href=\"https://www.bbmri.de/news-events/article/?tx_news_pi1%5Bnews%5D=366&tx_news_pi1%5Bcontroller%5D=News&tx_news_pi1%5Baction%5D=detail&cHash=43adadb762a27952c1b46b4081f0ca4e\" target=\"_blank\">German Biobank Node (GBN)</a> on the aims of GHGA (in German only)"
+    },
     "item2": {
+      "title": "June 29, 2020 - Press Release about GHGA",
+      "content": "GHGA was selected for funding as part of the German NFDI Initiative. Please see the press releases by <a href=\"https://www.dkfz.de/de/presse/pressemitteilungen/2020/dkfz-pm-20-41c-Genomdaten-besser-nutzen-Konsortium-entwickelt-die-notwendige-Dateninfrastruktur.php\" target=\"_blank\">DKFZ</a>, <a href=\"https://uni-tuebingen.de/universitaet/aktuelles-und-publikationen/attempto-online/forschung/newsfullview-attempto-forschung/article/konsortium-entwickelt-infrastruktur-fuer-genomdaten/\" target=\"_blank\">Tuebingen University</a> and <a href=\"https://www.embl.org/news/lab-matters/human-genome-phenome-archive/\" target=\"_blank\">EMBL</a>"
```
Comment (vinjana) [inline assets/locales/en/translation.json:44]:
```suggestion
      "content": "GHGA was selected for funding as part of the German NFDI initiative. Please see the press releases by <a href=\"https://www.dkfz.de/de/presse/pressemitteilungen/2020/dkfz-pm-20-41c-Genomdaten-besser-nutzen-Konsortium-entwickelt-die-notwendige-Dateninfrastruktur.php\" target=\"_blank\">DKFZ</a>, <a href=\"https://uni-tuebingen.de/universitaet/aktuelles-und-publikationen/attempto-online/forschung/newsfullview-attempto-forschung/article/konsortium-entwickelt-infrastruktur-fuer-genomdaten/\" target=\"_blank\">Tuebingen University</a> and <a href=\"https://www.embl.org/news/lab-matters/human-genome-phenome-archive/\" target=\"_blank\">EMBL</a>"
```

File: assets/locales/en/translation.json [hunk 4]
```diff
@@ -27,15 +27,39 @@
   "news": {
     "navTitle": "News",
     "title": "News",
-    "item1": {
+    "item5": {
+      "title": "November 19, 2020 - Join the GHGA Team",
+      "content": "We are constantly looking for talents to reinforce our GHGA Team. Please find current job offers under <a href=\"#jobs\">jobs</a>."
```
Comment (vinjana) [inline assets/locales/en/translation.json:32]:
```suggestion
      "content": "We are constantly looking for talents to reinforce our GHGA team. Please find current job offers under <a href=\"#jobs\">jobs</a>."
```

File: assets/locales/en/translation.json (hunk 1 above)
Comment (eufjan) [reply assets/locales/en/translation.json:56]: perfect

**Review by vinjana — APPROVED** [review]

**Comment by vinjana** [comment]
Hi @eufjan . I missed (like you) that for adding a new item to the menus currently a second page needs to be changed. The "Events" topic has not been added to `data/nav.html`. So when you go on "DATA" you won't see the "EVENTS".

**Comment by eufjan** [comment]
> 
> 
> Hi @eufjan . I missed (like you) that for adding a new item to the menus currently a second page needs to be changed. The "Events" topic has not been added to `data/nav.html`. So when you go on "DATA" you won't see the "EVENTS".

Hi @vinjana thanks, will update that in the next round. I want to write some basic documentaion. Should I put that into README.md ?

**Comment by vinjana** [comment]
Depends on the docu? Is it a lot? Github also has a Wiki feature. So if you think you need more structure, that may be a good alternative.

**Pull request merged**